src/qml/qgismobileapp.qml

```diff
@@ -931,16 +931,35 @@ ApplicationWindow {
     }
   }
 
+  NavigatingDrawer{
```

Comment:
(also, don't forget the space before the {)

```suggestion
  InformationDrawer {
```

Reply:
sure, additionally, it would be great if we could enable auto-formatting. As it stands, I've had to disable it to avoid interfering with other parts of the project.

Reply:
@mohsenD98 , enabling auto formating is on the wishlist :) we tried in the past but it had deficiencies. That was before Qt 6, it's probably much better now. A nice mission for another day :)